package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "nodejs_mp",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "@babel/cli": "^7.13.14",
+    "@babel/core": "^7.13.14",
+    "@babel/node": "^7.13.13",
+    "@babel/polyfill": "^7.12.1",
+    "@babel/preset-env": "^7.13.12",
+    "@hapi/joi": "^17.1.1",
+    "express": "^4.17.1",
+    "express-joi-validation": "^5.0.0",
+    "typescript": "^4.2.3",
```

Comment:
If you decide to use `babel`, I think `typescript` is an unnecessary dependency, isn't it? :)